src/styleguide/_patterns/02-molecules/02-locations/location-card.mustache

```diff
@@ -0,0 +1,17 @@
+<div class="grav-c-location-card">
+  <div class="grav-c-location-card__img">
+    <img src="{{location.img}}" width="425" height="300" alt="">
+  </div>
+  <div class="grav-c-location-card__content">
+    <h2>{{location.name}}</h2>
```

Comment:
what's the difference between `{{name}}` and `{{city}}` ?

Reply:
The main reason for this is New York/Brooklyn. We call it New York but the address uses Brooklyn where there would usually be a city